apis/user-java/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+springfox.documentation.swagger.v2.path=/api-docs
```

Comment:
docs base path should be `/api/docs/userjava`

Reply:
As discussed this, breaks things when changed and will need more digging - so shelving for now.